engine/src/main/java/com/google/android/fhir/search/MoreSearch.kt

```diff
@@ -149,3 +172,37 @@ val Order?.sqlString: String
       Order.DESCENDING -> "DESC"
       null -> ""
     }
+
+fun getConditionParamPair(
+  prefix: ParamPrefixEnum?,
+  value: BigDecimal,
+  precision: BigDecimal
+): Pair<String, List<Double>> {
+  return when (prefix) {
+    ParamPrefixEnum.EQUAL, null ->
+      "index_value >= ? AND index_value < ?" to
+        listOf((value - precision).toDouble(), (value + precision).toDouble())
+    ParamPrefixEnum.GREATERTHAN -> "index_value > ?" to listOf(value.toDouble())
```

Comment:
```suggestion
    ParamPrefixEnum.GREATERTHAN -> "index_value > ?" to listOf((value - precision).toDouble())
```

same should be applied to everything below

Reply:
according to the [standard](https://www.hl7.org/fhir/search.html#number)

> When a comparison prefix in the set lgt, lt, ge, le, sa & eb is provided, the implicit precision of the number is ignored, and they are treated as if they have arbitrarily high precision

Reply:
oh i didn't realise this is described in the standard. can you maybe add a comment in the code to link to https://www.hl7.org/fhir/search.html#number?